addons/account/models/account_journal_dashboard.py

```diff
@@ -848,12 +848,21 @@ def action_create_new(self):
             'context': self._get_move_action_context(),
         }
 
+    def _build_no_journal_error_msg(self, company_name, journal_types):
+        return _(
+                "No journal could be found in company %(company_name)s for any of those types: %(journal_types)s",
```

Comment:
adapt this in .pot

Reply:
Indeed, the model of the traduciton is now account_journal and not account_move anymore forgot that, mb :+1:

Reply:
> Indeed, the model of the traduciton is now account_journal and not account_move anymore forgot that, mb 👍 (line 9530 in addons/account/i18n/account.pot)

Got it 😄